_rules/SC1-2-1-audio-only-alternative.md

```diff
@@ -0,0 +1,112 @@
+---
+name: audio only has a text alternative
+rule_type: composite
+description: |
+  This rule checks if audio only elements have a text alternative available
+
+success_criterion:
+- 1.2.1 # Audio-only and Video-only (Prerecorded)
+
+atomic_rules:
+- SC1-2-1-audio-transcript
+- SC1-2-1-media-alternative-audio
+
+authors:
+- Wilco Fiers
+- Brian Bors
+- John Hicks
+- Rafal Charlampozicz
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button".  If a play button exists the rule applies if the button is [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
```

Comment:
The second sentence doesn't parse grammatically. I think this should be either one sentence:

> The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button" that is [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies).

Or as a list:

> The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is:
> - playing; or,
> - has a "play button" that is [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies).